index.html

```diff
@@ -0,0 +1,181 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8">
+		<meta name="viewport" content="width=device-width, initial-scale=1.0">
+		<link rel="stylesheet" href="styles/main.css">
+		<title>Dog Adoption | Home</title>
+	</head>
+	<body>
+		<header>
+			<div class="container">
+				<div class="title">
+					<div class="left-side">
+						<img src="img/logo.svg" class="logo-img" alt="">
+						<p class="logo-text">Seattle's premiere dog adoption agency</p>
+					</div>
+					<div class="right-side">
+						<img src="img/cart-icon.svg" class="cart-img" alt="">
+						<span class="cart-price">$1234.56</span>
+						<a href="checkout.html" class="button-top">Checkout</a>
+					</div>
+				</div>
+			</div>
+		</header>
+		<nav class="nav-first">
+			<div class="container">
+				<a href="index.html">Home</a>
+				<span class="after"></span>
+				<a href="dogs.html">Dogs</a>
+				<span class="after"></span>
+				<a href="blog.html">Blog</a>
+			</div>
+		</nav>
+		<section class="home-second">
+			<div class="container">
+				<div class="main-body">
+					<div class="block-1">
+						<div class="first-side">
+							<img src="img/home-intro.jpg" alt="">
+							<img src="img/home-intro2.jpg" alt="">
+						</div>
+						<div class="second-side">
+							<h2>Adopt a Dog</h2>
+							<p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with </p>
+						</div>
+					</div>
+					<div class="block-2">
+						<div class="item">
+							<img src="img/item-1.jpg" alt="" onclick="imgClickMurphy()">
+							<div class="mini-container">
+								<h3>Murphy</h3>
+								<span class="bold">Cost to Adopt</span>
+								<span class="price">$ 123.45</span>
+								<p class="text">Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry</p>
+								<button class="button" onclick="adoptClick()">Adopt</button>
```

Comment:
Think of how you could pass in an adoption price? What if all the adoption fees were different?